Test/Test Utilities/TestUtilities.swift

```diff
@@ -165,16 +165,14 @@ class AblyTests {
         return protocolMessage
     }
 
-    class func newPresenceProtocolMessage(_ channel: String, action: ARTPresenceAction, clientId: String) -> ARTProtocolMessage {
+    class func newPresenceProtocolMessage(id: String, channel: String, action: ARTPresenceAction, clientId: String, connectionId: String) -> ARTProtocolMessage {
```

Comment:
no need to provide `id`

Reply:
This actually comes under idempotent publishing. Mostly `id` is assigned to message by `ably` itself. 
There are limited usecases where client provide `id` by itself. 
You can check
https://faqs.ably.com/client-specified-message-id-restrictions-for-multiple-messages-published-atomically

Reply:
You can keep it if needed, otherwise for all realtime publish calls, we don't really include `messageid` as such